src/autocomplete.css

```diff
@@ -20,17 +20,17 @@
 }
 
 .autocomplete__hint {
-  color: #BFC1C3;
+  color: #b1b4b6;
   position: absolute;
 }
 
-.autocomplete__input--default{
-  padding: 4px;
+.autocomplete__input--default {
+  padding: 5px;
```

Comment:
Should the autocomplete have a `height: 40px` to match our text inputs? At the minute it's 39px which is frustratingly close.

Reply:
I've tried not to change that since this component relies on layering of inputs and overlays for some features and it felt risky to change, so wanted to do a simple update and maybe do something better if we decide to do the stuff Dave was proposing in the other PR.

Reply:
I'd do it :shipit:

Reply:
This pull request isn't meant to be making the component 'proper GOV.UK Frontend' so would be nice to get this in and improve it later in the way Dave has suggested, will see what Ollie thinks 😬